modules/libevdev/1.13.2/MODULE.bazel

```diff
@@ -0,0 +1,6 @@
+module(
+    name = "libevdev",
```

Comment:
you need to have `version = "1.13.2",` here (also for the patch; and don't forget to fix up the integrity values)

Reply:
Done.